src/utilities/data-model.js

```diff
@@ -44,7 +44,7 @@ function savePhoto(geoHash, filePath) {
         console.log(err.message);
       });
 
-    var large = resizePhoto(filePath, 200, 200, 30)
+    var large = resizePhoto(filePath, 400, 500, 60)
```

Comment:
What would be a good directory + file to put that in?  Not sure where it should fall under.